src/main/java/com/marklogic/mgmt/admin/AdminManager.java

```diff
@@ -189,19 +190,21 @@ public void setSslFipsEnabled(final boolean enabled, final int appServicesPort)
         invokeActionRequiringRestart(new ActionRequiringRestart() {
             @Override
             public boolean execute() {
-                RestTemplate rt = RestTemplateUtil.newRestTemplate(adminConfig.getHost(), appServicesPort,
-                        adminConfig.getUsername(), adminConfig.getPassword());
+				RestConfig evalConfig = new RestConfig(adminConfig);
+				evalConfig.setPort(appServicesPort);
+
                 HttpHeaders headers = new HttpHeaders();
                 headers.setContentType(MediaType.APPLICATION_FORM_URLENCODED);
                 MultiValueMap<String, String> map = new LinkedMultiValueMap<>();
                 map.add("xquery", xquery);
                 HttpEntity<MultiValueMap<String, String>> entity = new HttpEntity<>(map,
 					headers);
-                String url = format("http://%s:%d/v1/eval", adminConfig.getHost(), appServicesPort);
+
                 if (logger.isInfoEnabled()) {
                     logger.info("Setting SSL FIPS enabled: " + enabled);
                 }
-                rt.exchange(url, HttpMethod.POST, entity, String.class);
+				URI url = evalConfig.buildUri("/v1/eval");
```

Comment:
Switched this method to build its URI dynamically instead of hardcoding "http" usage.